### PR TITLE
Update dependencies of local charts when running 'sync'

### PR DESCRIPTION
**Pull request by cmeury**
As discussed in #26.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ COMMANDS:
      repos   sync repositories from state file (helm repo add && helm repo update)
      charts  sync charts from state file (helm upgrade --install)
      diff    diff charts from state file against env (helm diff)
-     sync    sync all resources from state file (repos && charts)
+     sync    sync all resources from state file (repos, charts and local chart deps)
```
Comment (mumoshu) [inline README.md]: Thanks for also updating docs! I miss it often 😄

File: helmexec/exec.go [hunk 2]
```diff
@@ -90,8 +84,12 @@ func (helm *execer) exec(args ...string) ([]byte, error) {
 	if helm.kubeContext != "" {
 		cmdargs = append(cmdargs, "--kube-context", helm.kubeContext)
 	}
+	helm.write([]byte(fmt.Sprintf("exec: helm %s\n", strings.Join(cmdargs, " "))))
+	return helm.runner.Execute(command, cmdargs)
+}
+
+func (helm *execer) write(out []byte) {
```
Comment (mumoshu) [inline helmexec/exec.go]: I love this.

**Comment by mumoshu** [comment]
LGTM 👍 Thanks as always for your contribution!

**Comment by cmeury** [comment]
Would definitely be good if someone could test this a bit more thoroughly. I did a few basic test runs and otherwise rely on the unit tests. Shouldn't be too fragile, but I wonder whether there's some glaring omission.

**Pull request merged**